src/pybind/mgr/rgw/__init__.py

```diff
@@ -1,5 +1 @@
-# flake8: noqa
-try:
-    from .module import Module
-except ImportError:
```

Comment:
for my info, why do we no longer need this try-except block?

Reply:
> for my info, why do we no longer need this try-except block?

TBH I'm not sure why the try-catch was there. Module is normally defined for mgr modules and it's the case for this rgw module so I don't see the need for the try-catch.